packages/client/components/PaletteColor/PaletteColor.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import useHover from '../../hooks/useHover'
+import styled from '@emotion/styled'
+
+interface Props {
+  color: string
+  isPicked: boolean
+  currentSelection: boolean
+}
+
+interface ColorProps extends Props {
+  isHover: boolean
+}
+
+const ColorItem = styled('li')<ColorProps>(({isHover, color, isPicked, currentSelection}) => ({
+  backgroundColor: color,
+  cursor: isHover ? 'pointer' : 'grab',
+  height: 32,
+  width: 32,
+  borderRadius: '50%',
+  opacity: isPicked && !currentSelection ? 0.25 : 1,
```

Comment:
Yeah I can see that. On the figma, theres an alternate design that assigns a minus symbol when a color (other than the currently selected one) has been chosen. I think this could be built upon so that when a user hovers over a color, a plus symbol shows. This would help to indicate that selected colors can still be available to choose. 

Another option would be to keep the opacity as I have it now, but when the user hovers over a selected color, the opacity goes back to 1 so they can view the original color that they could select.